src/FormBuilder/resources/js/form-builder/src/containers/BlockEditorContainer.tsx

```diff
@@ -10,12 +10,19 @@ import {__} from "@wordpress/i18n";
 
 
 /**
+ * @unreleased Early return dispatchFormBlocks if there are no unwrapped blocks.
  * @since 3.0.0
  */
 export default function BlockEditorContainer() {
     const {blocks} = useFormState();
     const dispatch = useFormStateDispatch();
     const dispatchFormBlocks = (blocks) => {
+        const hasUnwrappedBlocks = blocks.some((block) => block.name !== 'givewp/section');
+
+        if (!hasUnwrappedBlocks) {
+            return;
+        }
```

Comment:
@pauloiankoski nice find! I wonder why this matters? 

However, we will still need to dispatch the blocks in order for the block schema to persist into state or else you won't be able to actually update the form.  I think this is all we would need to do:

```suggestion
        if (!hasUnwrappedBlocks) {
            return dispatch(setFormBlocks(blocks));
        }
```

Reply:
Implemented on 90970eca9a0345bbaf636b4d5e363cbc257a5ec6

Reply:
Cool cool, actually should we be returning this or breaking it out into an if / else?  I don't believe the prop callbacks this is used by are expecting to receive anything.

Reply:
```js
updateParent( blocks, {
	selection: {
		selectionStart: getSelectionStart(),
		selectionEnd: getSelectionEnd(),
		initialPosition:
			getSelectedBlocksInitialCaretPosition(),
	},
	undoIgnore,
} );
```
Where `updateParent` is a variable where the callback function is stored. It means the return of that callback function is never used, so it's like `void` is expected.